data_sources/cisco_secure_firewall_threat_defense_connection_event.yml

```diff
@@ -0,0 +1,110 @@
+name: Cisco Secure Firewall Threat Defense Connection Event
+id: 18878597-8f8a-4bca-a805-bfbe35e00032
+version: 1
+date: '2025-04-01'
+author: Nasreddine Bencherchali, Splunk
+description: Data source object for raw connection events from Cisco Secure Firewall Threat Defense
+source: not_applicable
+sourcetype: cisco:sfw:estreamer
+supported_TA:
```

Comment:
@nasbench :
@MHaggis Can we add some output_fields to the Crush FTP data source file? 

Example: https://github.com/splunk/security_content/blob/1a084095d53087f2535894becfefd668ecc8db9f/data_sources/sysmon_eventid_1.yml#L120

These fields is the minimum number of basic fields needed in an output of a detection.